env_variables.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+export BASE_URL=http://192.168.0.13:8888/quicksite/
+export BROWSER=chrome
+export RESULTS_DIR=$(pwd)/results
+export DB_PORT_OVERRIDE=8889
+export DB_HOST_OVERRIDE=localhost
+export DB_DATABASE_OVERRIDE=quicksitedb
+export DB_TABLE_PREFIX_OVERRIDE=wp_
+
+export WOO_KEY=ck_3be63c98d1de6a67334a63469f81b385a26e0a32
```

Comment:
dont check in your credentials. Use a different variables file for the credentials and don't check it in. 
Where ever you run your code GitLab/Jenkins you will set credentials as variables. you don't want to check in real credentials.

Reply:
change has made.